extensions/src/tables/index.ts

```diff
@@ -0,0 +1,326 @@
+import { ArgumentType, ButtonBlock, BlockType,
+  Block, DefineBlock, DynamicMenuThatAcceptsReporters,
+  Environment, Extension, ExtensionMenuDisplayDetails,
+  MenuItem, SaveDataHandler
+} from "$common";
+
+/**
```

Comment:
I think as a rule of thumb, we can/should delete these documentation comments from `index.ts` files before they are pushed to dev.